packages/react/src/constants/CardPropTypes.js

```diff
@@ -768,7 +768,9 @@ export const CardPropTypes = {
   resizeHandles: PropTypes.array,
   /** Optional callback function that is passed an onChange function and the original cardConfig object.
    * This allows additional information to be passed to be used in the Card Editor for this type.
-   * You need to return an array of child objects with a header: {title, tooltip: {tooltipText: PropTypes.string}} and content element to render * */
+   * You need to return an array of child objects with a header: {title, tooltip: {tooltipText: PropTypes.string}} and content element to render
+   * We recommend doing this dynamically with the CardEditor onRenderCardEditForm property rather than hardcoding it in the Card JSON
```

Comment:
Not sure I understand this. So are we going to keep two different approaches for doing the same thing? Is it not possible for you to use onRenderCardEditForm and still add the modifications somehow?

Reply:
Yes there are two ways, we basically we preserve the old static prop (for non-breaking changes) but recommend using the dynamic prop, was the intention with the comment.  We don't want to deprecate the old prop (because we need it to actually use the dynamic prop) but we would probably want to hide it from downstream consumers in a future release.

Reply:
I see, thanks for the clarification.